pkg/network/master/subnets.go

```diff
@@ -210,20 +191,27 @@ func (master *OsdnMaster) watchNodes() {
 
 func (master *OsdnMaster) handleAddOrUpdateNode(obj, _ interface{}, eventType watch.EventType) {
 	node := obj.(*kapi.Node)
-	nodeIP, err := getNodeIP(node)
-	if err != nil {
-		glog.Errorf("Failed to get node IP for node %s, skipping %s event, node: %v", node.Name, eventType, node)
+
+	var nodeIP string
+	for _, addr := range node.Status.Addresses {
+		if addr.Type == kapi.NodeInternalIP {
+			nodeIP = addr.Address
+			break
+		}
+	}
+	if len(nodeIP) == 0 {
+		glog.Errorf("Node IP is not set for node %s, skipping %s event, node: %v", node.Name, eventType, node)
```

Comment:
This entire file needs to use utilruntime.HandleError.  Any kube code that eats an error (that uses glog.Errorf) should be utilruntime.HandleError instead.  That ensures that we get rate limiting, that errors are reported to sentry, and also that hotloop requests get processed.

Can be a follow up PR but please correct that before we ship 3.9

Reply:
Created follow up pr: https://github.com/openshift/origin/pull/18343